mappings/net/minecraft/client/render/block/entity/BlockEntityRenderer.mapping

```diff
@@ -5,9 +5,9 @@ CLASS net/minecraft/class_827 net/minecraft/client/render/block/entity/BlockEnti
 	METHOD method_3563 rendersOutsideBoundingBox (Lnet/minecraft/class_2586;)Z
 		ARG 1 blockEntity
 	METHOD method_3569 render (Lnet/minecraft/class_2586;FLnet/minecraft/class_4587;Lnet/minecraft/class_4597;II)V
-		ARG 1 blockEntity
-		ARG 2 tickDelta
+		ARG 1 entity
+		ARG 2 yaw
```

Comment:
Seriously, what is the yaw of a block entity?
```suggestion
		ARG 2 tickDelta
```

Reply:
Oops. Checked in the skull renderer, and it is indeed the tickDelta.